.github/workflows/release.yml

```
@@ -122,7 +126,10 @@ jobs:
             NEXT_VERSION=$(python bump.py $RELEASE_TYPE $RELEASE_VERSION)
         elif [[ $RELEASE_TYPE == "minor" ]]
         then 
-            NEXT_VERSION=$(python bump.py $RELEASE_TYPE $RELEASE_VERSION)          
+            NEXT_VERSION=$(python bump.py $RELEASE_TYPE $RELEASE_VERSION)
+        elif [[ $RELEASE_TYPE == "major" ]]      
+        then
+            NEXT_VERSION=$(python bump.py "minor" $RELEASE_VERSION)
```

I have no idea if this is right or wrong, but this just caught my eye. Since release type is "major" here, shouldn't the bump.py be sent in "major" as its argument value?
```suggestion
            NEXT_VERSION=$(python bump.py "major" $RELEASE_VERSION)
```

---

For that case, which is calculate the next version, the bump shouldn't be to 3.0, but to 2.1 instead, for that reason I'm setting a minor bump there.